target/linux/mvebu/image/Makefile

```diff
@@ -13,8 +13,10 @@ include $(INCLUDE_DIR)/image.mk
 
 KERNEL_LOADADDR := 0x00008000
 
+SIGNATURE:=$(shell perl -e 'printf("%08x", rand(0xFFFFFFFF))')
```

Comment:
This will break reproducible builds, can you generate it from SOURCE_DATE_EPOCH for example

Reply:
Good point, I'll change that. This was taken from x86.

Reply:
This is now generated from SOURCE_DATE_EPOCH as recommended.